### DISCUSSION TITLE
Millfork NES game (unfinished)

### DISCUSSION THREAD

**Discussion by joshop** [KarolS/millfork#139] [Show and tell]
[https://github.com/joshop/brimstone](https://github.com/joshop/brimstone) - it's nowhere close to finished, but it's mostly in Millfork (about 1500 lines of code). Targeting 32K of PRG space and 8K of CHR.

**Comment by agg23** [comment]
Just in case people are interested, I'll add my derpy projects to the pile

* https://github.com/agg23/FINes - Aquarium simulator. "Finished" game for the NESdev 2020 competition. Ended up being much more limited and unimpressive due to spending most of the month fighting with compiler bugs (most of which I never tracked down)
* https://github.com/agg23/dungeon-generator - Demonstration of a simple room generation algorithm, with a corresponding generator in JS

**Reply by joshop** [reply]
Nice to see someone else using Millfork for NESdev. There are a couple compiler bugs that popped up in mine too (that I had to work around with assembly). Maybe I'll finish that game one day.
